Assets/Scripts/LevelEditor/LevelEditor.cs

```diff
@@ -125,22 +152,43 @@ private void NewLevel()
         newLevel.gravityMultiplier = 1;
 
         string currentTime = DateTime.Now.ToString("MM-dd-yyyy_hh-mm-ss-FFF");
-        string scriptableObjectPath = Path.Combine(Application.persistentDataPath, currentTime + "scriptableObject.level");
+        string scriptableObjectPath = Path.Combine(levelEditorFolderPath, currentTime + ".level");
         newLevel.levelEditorScriptableObjectPath = scriptableObjectPath;
 
-        string folder = Path.Combine(Application.persistentDataPath, currentTime);
-        newLevel.levelEditorFolder = folder;
+        newLevel.levelEditorFolder = levelEditorFolderPath;
+
+        string levelDataPath = Path.Combine(levelEditorFolderPath, currentTime + ".json");
+        newLevel.levelEditorLevelDataPath = levelDataPath;
 
-        string levelDataPath = Path.Combine(folder, "levelData.json");
-        newLevel.levelEditorScenePath = levelDataPath;
+        if (!Directory.Exists(levelEditorFolderPath))

```

Comment:
when you add them to their own folder, you'll have to make sure the parent folder exists as well